packages/sanity/src/core/store/_legacy/history/useTimelineStore.ts

```diff
@@ -194,7 +194,6 @@ export function useTimelineStore({
       findRangeForSince: (chunk: Chunk) => controller.findRangeForNewSince(chunk),
       loadMore: () => {
         controller.setLoadMore(true)
-        timelineStateRef.current.isLoading = true
```

Comment:
I noticed this in some cases was blocking the ui in a loading state, which never ended because the load more action was not actually fetching new data, so the controller was not triggering any update, so the `isLoading` returned in line 220 was never resetting to false.